### PR TITLE
Added swagger test for Tokens by code 

### PR DESCRIPTION
**Pull request by shoebkhan09**

### CODE REVIEW

File: oxd-server/src/main/resources/swagger.yaml [hunk 1]
```diff
@@ -954,13 +954,17 @@ paths:
               - oxd_id
               - refresh_token
               - scope
+              - protection_token
```
Comment (yuriyz) [inline oxd-server/src/main/resources/swagger.yaml]: why do we need it here? `protection_access_token` already goes via `Authorization` header.

Comment (shoebkhan09) [reply oxd-server/src/main/resources/swagger.yaml]: @yuriyz , I cross checked with server side code, and as you suggest, it is not necessary to set the access token in request parameters for this operation. Let me know if we should remove it from both existing and swagger tests.

Comment (yuriyz) [reply oxd-server/src/main/resources/swagger.yaml]: In socket connection there are no header, so we pass it explicitly as parameter. In HTTP we have authorization header. Why do we need to duplicate it? Please remove it. In chat I meant that this command must be protected, and if it is not then add protection. I have just checked and it is protected.

**Comment by shoebkhan09** [comment]
@yuriyz , will you please review ?

**Pull request merged**